src/amo/store.js

```diff
@@ -43,6 +44,7 @@ export default function createStore(initialState = {}) {
       installations,
       landing,
       reduxAsyncConnect,
+      redirectTo,
```

Comment:
It would be good for sanity to also add this to the Discopane. We won't need it there but since Discopane also runs from `server/base.js` it will make sure that `store.getState().redirectTo` is never undefined.

Reply:
ah that makes sense, even though I think there is a check for that.